### DISCUSSION TITLE
Is there a way to use markdown inside a note (or a grid)?

### DISCUSSION THREAD

**Discussion by aborruso** [observablehq/framework#862] [Q&A] [answered]
A note is HTML. Is there a way to use markdown formatted text inside a note?

Thank you

```
<div class="note">
This is a note.
</div>
```

**Comment by Fil** [comment, marked as answer]
no, not at this point

**Reply by aborruso** [reply]
@Fil and is it the same for grid? I think yes

**Reply by mbostock** [reply]
Correct. This is a property of the [CommonMark specification](https://spec.commonmark.org/0.30/#html-blocks) combined with the fact that Framework doesn’t currently allow blank lines within HTML blocks.
